projects/tuvali.md

```diff
@@ -0,0 +1,87 @@
+# Project Name
+Tuvali - Means a tunnel in [Tamil](https://en.wikipedia.org/wiki/Tamil_language)
+
+# Preferred Maturity Level
+Lab
+
+# Project Description
+This library contains the implementation for Android & IOS for the Openid4BLE specification. It's written in Kotlin & Swift.
```

Comment:
Can you provide the link to the specification? If it refers to https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-over-ble-1_0.html, can you update the description to specify its presentation over BLE? Currently, it just says OpenID for BLE.

Reply:
Well, I see Verifiable Presentation (VP) mention in the next section. It would be helpful to specify in the description section as well.